leaderboardFeature/src/main/java/com/example/leaderboardfeature/model/Leaderboard.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.leaderboardfeature.model
+
+import java.util.Date
+
+data class Leaderboard(
+  val imageId: Int,
+  val username: String,
+  val score: Int,
+  val date: Date
+)
```

Comment:
This need to be in Domain layer but for now I want to leave it here and when viewModel will be used we can transfer this file

Reply:
`date` is confusing name, use different more screaming naming

Reply:
> 

Well in this case I wanted to not load images from network, it will be predownloaded images so we can use them. I forgot to mention it in the comments